jekyll/_cci2/optimizations.md

```diff
@@ -27,109 +27,48 @@ Choosing the right docker image for your project can have huge impact on build t
+Caching should be one of the first things you consider when trying to optimize your jobs. If a job fetches data at any point, it is likely that you can make use of caching. A common example is the use of a package/dependency manager. If your project uses Yarn, Bundler, or Pip, for example, the dependencies downloaded during a job can be cached for later use rather than being re-downloaded on every build.
 
-Consult the [Caching]({{site.baseurl}}/2.0/caching) page to learn more.
+* Read more on the [Caching Dependencies]({{site.baseurl}}/2.0/caching) page.
 
 ## Workflows
 {: #workflows }
 
-Workflows provide a means to define a collection of jobs and their run order. If at any point in your build you see a step where two jobs could happily run independent of one another, workflows may be helpful. Workflows also provide several other features to augment and improve your build configuration. Read more about workflows in the [workflow documentation]({{site.baseurl}}/2.0/workflows/).
-
-**Note**: Workflows are available to all plans, but running jobs concurrently assumes that your plan provides multiple machines to execute on.
-
-```yaml
-version: 2.1
-jobs: # here we define two jobs: "build" and "test"
-  build:
-    docker: # the docker executor is used
-      - image: cimg/<language>:<version TAG> # An example docker image
-        auth:
-          username: mydockerhub-user
-          password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
-    steps:
-      - checkout # Pulls code down from your VCS
-      - run: <command> # An example command
-  test:
-    docker: # same as previous docker key.
-      - image: cimg/<language>:<version TAG>
-        auth:
-          username: mydockerhub-user
-          password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
-    steps:
-      - checkout
-      - run: <command>
-workflows: # Here we can orchestrate our jobs into a workflow
-  version: 2
-  build_and_test: # A single workflow named "build_and_test"
-    jobs: # we run our `build` job and `test` job concurrently.
-      - build
-      - test
-```
-
-You can view more examples of workflows in the  [CircleCI demo workflows repo](https://github.com/CircleCI-Public/circleci-demo-workflows/).
+Workflows provide a means to define a collection of jobs and their run order. If at any point in your build you see a step where two jobs could happily run independent of one another, workflows may be helpful. Workflows also provide several other features to augment and improve your build configuration. Read more about workflows on the [Workflow]({{site.baseurl}}/2.0/workflows/) page.
+
+* You can view examples of workflows in the  [CircleCI demo workflows repo](https://github.com/CircleCI-Public/circleci-demo-workflows/).
 
 ## Workspaces
 {: #workspaces }
 
-**Note**: Using workspaces presumes that you are also using [workflows](#workflows).
+**Note**: Using workspaces presumes that you are also using [Workflows](#workflows).
```

Comment:
We can remove this line. I think we can now assume everyone uses workflows anyway.